tracks/python/exercises/kindergarten-garden/mentoring.md

```diff
@@ -0,0 +1,36 @@
+### Reasonable solutions
+
+```python
+class Garden(object):
+
+    DEFAULT_STUDENTS = [
+        'Alice', 'Bob', 'Charlie', 'David',
+        'Eve', 'Fred', 'Ginny', 'Harriet',
+        'Ileana', 'Joseph', 'Kincaid', 'Larry',
+    ]
+
+    PLANT_NAMES = {
+        'C' : 'Clover',
+        'G' : 'Grass',
+        'R' : 'Radishes',
+        'V' : 'Violets'
+    }
+
+    def __init__(self, diagram, students=DEFAULT_STUDENTS):
```

Comment:
I would not accept a solution that used a _mutable_ default value; any mutation that happens at any point to the default will affect all future calls that use the default.

The usual idiom would be to do:

```python
def __init__(self, diagram, students=None):
    self.students = sorted(students if students is not None else self.DEFAULT_STUDENTS)
```